analytics/src/analytics/integrations/etldb/migrations/versions/0000_drop_tables_etldb.sql.ignore

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS schema_version;
```

Comment:
Did you mean to leave this file in? I don't understand what it's for

Reply:
Yes, it's useful during dev. Notice the `.ignore` suffix, which causes the `init-db` command to ignore it

Reply:
Having said that, I can remove it if it raises concerns. LMK

Reply:
I removed it out of an abundance of caution